src/components/activities.js

```diff
@@ -282,6 +282,7 @@ export class ActivityPorts {
         'publicationId': pageConfig.getPublicationId(),
         'productId': pageConfig.getProductId(),
         '_client': 'SwG $internalRuntimeVersion$',
+        'supportsClientLogging': true,
```

Comment:
Which client is `supportsClientLogging` referring to?

Reply:
It is a flag to tell the server that the server can message log requests back to the client and that the client has the wiring to handle that logging.  This way when I re-write the server logging to message back to the client it can:

if (supports) -> message to client -> client messages to {clearcut, propensity}
else -> message to clearcut

You can actually see most of that change in a CL now (just sent to you), I just can't get the tests to run due to contribution iframe issues unrelated to my dev.

This is necessary because:
  1) On mondays we released swg-js which is immediately sent out to everyone not on AMP
  2) It takes 1 weeks to get promoted from the amp canary to amp prod
  3) There are additional references to old versions of swg in the AMP cache that can take weeks to actually update
  4) There are random old clients out there that I have yet to explain.  Probably because a publisher copied our code instead of referencing through Google properly.

Reply:
So I consider any JavaScript code running in the browser being on the "client side". Could we pick a more descriptive name for this flag? Something like, "supportsExternalLogging"?

Reply:
supportsEventManager